docs/content/engines/presto.md

```diff
@@ -0,0 +1,97 @@
+|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [0.236,0.268)    | [paimon-presto-0.236-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/paimon/paimon-presto-0.236/{{< version >}}/paimon-presto-0.236-{{< version >}}.jar) |
+| [0.268,0.273)    | [paimon-presto-0.268-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/paimon/paimon-presto-0.268/{{< version >}}/paimon-presto-0.268-{{< version >}}.jar) |
+| [0.273,0.279]    | [paimon-presto-0.273-{{< version >}}.jar](https://repo.maven.apache.org/maven2/org/apache/paimon/paimon-presto-0.273/{{< version >}}/paimon-presto-0.273-{{< version >}}.jar) |
+
+{{< /stable >}}
+
+{{< unstable >}}
+
+|     Version      | Jar                                                                                                                                                                                                            |
+|------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| [0.236,0.268)    | [paimon-presto-0.236-{{< version >}}.jar](https://repository.apache.org/snapshots/org/apache/paimon/paimon-presto-0.236/{{< version >}}/) |
+| [0.268,0.273)    | [paimon-presto-0.268-{{< version >}}.jar](https://repository.apache.org/snapshots/org/apache/paimon/paimon-presto-0.268/{{< version >}}/) |
+| [0.273,0.279]    | [paimon-presto-0.273-{{< version >}}.jar](https://repository.apache.org/snapshots/org/apache/paimon/paimon-presto-0.273/{{< version >}}/) |
+
+{{< /unstable >}}
+
+You can also manually build bundled jar from the source code.
+
+To build from source code, [clone the git repository]({{< github_repo >}}).
+
+Build bundled jar with the following command.
+
+```
+mvn clean install -DskipTests
+```
+
+You can find Presto connector jar in `./paimon-presto/paimon-presto-<presto-version>/target/paimon-presto-*.jar`.
+
+Then, copy `paimon-presto-*.jar and flink-shaded-hadoop-*-uber-*.jar` to plugin/paimon.
+
+## Configure Paimon Catalog
+
+Catalogs are registered by creating a catalog properties file in the etc/catalog directory. For example, create etc/catalog/paimon.properties with the following contents to mount the paimon connector as the paimon catalog:
+
+```
+connector.name=paimon
+warehouse=file:/tmp/warehouse
+```
+
+If you are using HDFS, choose one of the following ways to configure your HDFS:
+
+- set environment variable HADOOP_HOME.
+- set environment variable HADOOP_CONF_DIR.
+- configure fs.hdfs.hadoopconf in the properties.
+
+You can configure kerberos keytag file when using KERBEROS authentication in the properties.
+
+```
+security.kerberos.login.principal=hadoop-user
```

Comment:
Dose `paimon-presto` invoke `SecurityContext.install?`

Reply:
currently not calling

Reply:
so maybe this is not work now.

Reply:
yes, i'll do it later

Reply:
Added SecurityContext.install @JingsongLi